torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9575,11 +9575,18 @@ def __call__(self, opinfo, device, dtype, requires_grad, **kwargs):
         _foreach_inputs_kwargs["requires_grad"] = requires_grad
         _allow_higher_dtype_scalars = kwargs.pop("allow_higher_dtype_scalars", False)
 
-        for num_tensors, ord, out_dtype in product(
+        for num_tensors, ord, out_dtype, intersperse_empty_tensors in product(
```

Comment:
I'm not sure inputs here are large enough to trigger the bug. We need to test 2 cases
1) Enough chunks across all the tensors to trigger multiple kernel launches. That requires tensors totalling 65536 * 320 elements today, given that we launch at most 320 blocks
2) Enough tensors to trigger multiple kernel launches, don't know off the top of my head but should be at least 50-100
By looking at it, foreach inputs would generate at most 10 small-ish tensors, so might not trigger either case. 
We might be better off just writing one off tests for this, wdyt?

Reply:
Yea, that'd probably be easiest, the meta tests also just failed too so that makes the decision easier

Reply:
Eventually we probably want both tests